pkg/configuration/config.go

```diff
@@ -67,6 +69,26 @@ func (c *Config) GetConfigInfoStr(color bool) string {
 	sb.WriteString("VMs:\n")
 	sb.WriteString(c.getVMsInfoStr(color))
 
+	// Summary of IP Ranges
+	sb.WriteString(common.OutputSectionSep)
+	sb.WriteString("IP Ranges:\n")
+	sb.WriteString(c.getIPRangeInfoStr(color))
+
+	// External EndPoints
+	sb.WriteString(common.OutputSectionSep)
+	sb.WriteString("External EndPoints:\n")
+	sb.WriteString(c.getExternalEPInfoStr(color))
+
+	// Internal IP Ranges
+	sb.WriteString(common.OutputSectionSep)
+	sb.WriteString("Internal IP:\n")
+	sb.WriteString(c.getInternalEPInfoStr(color))
+
+	// Rule Blocks
+	sb.WriteString(common.OutputSectionSep)
+	sb.WriteString("Rule Blocks:\n")
+	sb.WriteString(c.getRuleBlocksStr(color))
+
```

Comment:
can be moved to a separate section of the report?
this displays inferred endpoints / ip ranges for connectivity analysis.. maybe it should be part of the connectivity analysis report? 
maybe consider a flag for `analyze` command to display this report (as output, not as verbose debug log) prior to the actual connectivity map?

Reply:
I think we need to have a deeper think:
In one hand, a report that will come with `analyze`, should also have the vm information.  on the other, I think that some of the info here is helpful for syntheses. 
https://github.com/np-guard/vmware-analyzer/issues/378